phases/ephemeral/witx/wasi_ephemeral_proc.witx

```diff
@@ -8,9 +8,12 @@
 (use "typenames.witx")
 
 (module $wasi_ephemeral_proc
-  ;;; Terminate the process normally. An exit code of 0 indicates successful
-  ;;; termination of the program. The meanings of other values is dependent on
-  ;;; the environment.
+  ;;; Terminate the process normally. An exit code of `$exitcode::success`
+  ;;; reports successful completion of the program. An exit code of
+  ;;; `$exitcode::failure` or any other value less than 126 reports a
+  ;;; failure, and the value is provided to the environment. If a value
+  ;;; of at least 126 is given, this function behaves as if it were
```

Comment:
I find the phrases "If a value of at least 126 is given" I little hard to parse.   Wouldn't "If a value of 126 or above..", or "If a value greater than or equal to 126.." be more a commonly used phasing.

I noticed this phrases on the PR description and was momentarily confused there too.   Is this choice of phrase deliberate, or following some precedent?

Reply:
Heh, I'm reading a book right now that uses "at least" and "at most" instead of "greater than or equal" and "less than or equal". It makes sense in that context, but this PR isn't in that context ;-). I'll change it.